cms/templates/partials/metadata-tiles.html

```diff
@@ -63,11 +63,12 @@
           <span class="text">{{ page.duration }}</span>
         </li>
       {% endif %}
-      <li>
-        <span class="title">FORMAT</span>
-        <span class="text">Online</span>
-      </li>
-
+      {% if page.format %}
```

Comment:
This condition might be unnecessary since the format is not going to be `null`.

Reply:
Addressed